src/components/layouts/hero/hero.tsx

```diff
@@ -31,7 +31,7 @@ const Hero = ({ title, description, image, buttons }: PropTypes) => (
           </MoleculeTextInteraction>
         </ContentText>
       </Grid>
-      <Grid lg={6} md={8} sm={12} xs={12}>
+      <Grid lg={6} md={5} sm={12} xs={12}>
```

Comment:
```suggestion
      <Grid lg={6} md={6} sm={12} xs={12}>
```
see my previous comment for reasoning behind this suggestion

Reply:
```suggestion
      <Grid lg={7} md={6} sm={12} xs={12}>
```
lg has to be 7 right?